plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,44 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
```

Comment:
Since we're nitpicking… nit.

Reply:
What does nit mean? I've forgotten to ask before.

Reply:
```-- This function shows a dialog with input fields
-- for entering information for page streaming from
-- an input page number.```

How's this?

Reply:
I'd keep it as brief as possible, but that would address the nit, yet. :-)
```
--- Shows a page number dialog for page streaming.
```

> What does nit mean? I've forgotten to ask before.

Fairly inconsequential things, though admittedly the usage is a bit odd because nits are head lice eggs and while small those are quite significant to find.

Reply:
I'll go for the briefer comment, Thanks.